server/repository/migrations/postgres/2022-03-15T10-00_report_table/up.sql

```diff
@@ -1,16 +1,21 @@
 CREATE TYPE report_type AS ENUM (
-    'OM_REPORT'
+    'OM_SUPPLY'
 );
 
 CREATE TYPE category_type AS ENUM (
```

Comment:
I think this should also change to context (report_context)

Reply:
good spot thanks [d834ce7](https://github.com/openmsupply/open-msupply/pull/147/commits/d834ce70a0a01c92189bd28d092da6e04da87758)